premailer/premailer.py

```diff
@@ -85,8 +85,28 @@ def _cache_parse_css_string(css_body, validate=True):
     """
     return cssutils.parseString(css_body, validate=validate)
 
+
+def _capitalize_property(match):
+    return '{0}:{1}{2}'.format(
+        match.group('property').capitalize(),
+        match.group('value'),
+        match.group('terminator'))
+
+
+def _capitalize_float_margin(css_body):
```

Comment:
Also, don't worry about polluting because define an [`__all__`](https://github.com/oasiswork/premailer/blob/43563fa932a9ccee04218556cbd82c30c6d72c9b/premailer/premailer.py#L38) anyway.